static/_redirects

```diff
@@ -1,5 +1,11 @@
 # Redirects
 
+/support/how-to/shut-down-a-cloud-server/  /how-to/shut-down-a-rackspace-cloud-server/  404
+/how-to/shut-down-a-rackspace-cloud-server/   /support/how-to/cloud-servers/shut-down-a-rackspace-cloud-server/  301 
+/support/how-to/shut-down-a-server/  /how-to/shut-down-a-rackspace-cloud-server/ 404
+/how-to/shut-down-a-rackspace-cloud-server/   /support/how-to/cloud-servers/shut-down-a-rackspace-cloud-server/  301 
+/support/how-to/shut-down-a-server-from-the-cloud-portal/  /how-to/shut-down-a-rackspace-cloud-server/  404 
+/how-to/shut-down-a-rackspace-cloud-server/   /support/how-to/cloud-servers/shut-down-a-rackspace-cloud-server/  301 
 /support/how-to/migrations-in-rackspace-cloud/  /how-to/migrations-in-the-rackspace-cloud/  404
 /how-to/migrations-in-the-rackspace-cloud/  /support/how-to/cloud-servers/migrations-in-the-rackspace-cloud/  301 
```

Comment:
I see this one's got the product name in the last URL too. Can you also remove that in line 10? Should be `/support-how-to/migrations-in-the-rackspace-cloud/  301`